bindings/generator/api/common.py

```diff
@@ -37,7 +41,7 @@ class VariantItemUnit:
 
 
 class VariantItemTuple:
-    def __init__(self, *items):
+    def __init__(self, *items: Any) -> None:
```

Comment:
are you sure `__init__`  is required to have a return type defined ?

Reply:
Mypy don't seems to require the return type defined in that case, I've removed it.